tests/ec2.rs

```diff
@@ -23,3 +24,47 @@ fn main() {
     }
 
 }
+
+#[test]
+// Test for issue 383
+fn dry_run() {
+    let credentials = DefaultCredentialsProvider::new().unwrap();
+    let ec2 = Ec2Client::new(credentials, Region::UsEast1);
+    let req = CreateSnapshotRequest {
+                     volume_id: "v-00000001".into(),
+                     description: None,
+                     dry_run: Some(true)
+                 };
+    match ec2.create_snapshot(&req) {
+        Ok(_) => {
+            panic!("create_snapshot should fail");
```

Comment:
Using should_panic resulted in a failure to match the single-quotes in the error message for the `query_serialization_name` test so I changed it to look for another portion of the message. I had no problems changing to should_panic for the `dry_run` test.